ios/iosHackthonApp/Scene Delegate/SceneDelegate.swift

```diff
@@ -11,9 +11,13 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
   func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
 
     let loginView = LoginView(viewModel: loginViewModel)
+    let appState = AppState()
```

Comment:
if we switch these two lines, could we then do
`let loginView = LoginView(viewmodel: loginViewModel).environmentObject(appState)`

or 
```
let loginView = LoginView(viewmodel: loginViewModel)
loginView.environmentObject(appState)
```

Something like this?

Reply:
yup..changed to 
`let appState = AppState()`
` let loginView = LoginView(viewModel: loginViewModel).environmentObject(appState)`